packages/SystemsThinkingWorkbench-Core.package/STWPlayfield.class/instance/addSimulationButtons.st

```diff
@@ -0,0 +1,15 @@
+initialization
+addSimulationButtons
+	
+	self playButton: STWButton new.
+	self playButton contents: 'Start'.
+	self playButton clickHandler: [ self toggleSimulation. ].
+	self addMorph: self playButton.
+	self playButton position: self position.
```

Comment:
I'd say we let the customer decide, as centering around the lower line is rather annoying and requires some rather complex calculations (for the two buttons).